### PR TITLE
Update/cte 88

### PR DESCRIPTION
**Pull request by giangdavis**
- Updated link management page to show link URL, name, status, updated at 
- Updated link management page to be paginated
- Updated modify link page to allow admins to modify link URL, name, and status 
- Updated add link page to specify a link URL and name 
- Updated add link form to automatically submit link as valid

### CODE REVIEW

File: resources/views/database/links.blade.php [hunk 1]
```diff
@@ -6,34 +6,62 @@
...
+        .table td {
+            text-align: center;
+        }
+
+        .pagination {
+            justify-content: center;
+        }
+    </style>
+
+@endpush
```
Comment (Dmitri-2) [inline resources/views/database/links.blade.php:67]: OK for now, but let's use the Bootstrap classes `text-center` and `d-flex justify-content-center` next time.

File: resources/views/database/links.blade.php [hunk 2]
```diff
@@ -6,34 +6,62 @@
...
-            </tbody>
-        </table>
+                </thead>
+                <tbody>
+                @foreach($links as $link)
+                    <tr>
+                        <th scope="row">{{$loop->index+1}}</th>
+                        <td>{{$link->name}}</td>
+                        <td>{{$link->linkText}}</td>
+                        <td>{{$link->status}}</td>
```
Comment (Dmitri-2) [inline resources/views/database/links.blade.php:27]: For the link status, let's put it into a Bootstrap pill: 
`<span class="badge badge-success">{{$link->status}}</span>`
and then adjust the color based on the status (`badge-success` for Valid, `badge-warning` for Unknown, and `badge-danger` for Broken.

File: resources/views/database/links.blade.php (hunk 1 above)
Comment (giangdavis) [reply resources/views/database/links.blade.php:67]: Yeah I was stuck on getting the table headers & data centered so I just used css, how can I do this? 
I managed to bootstrap the pagination though with `d-flex justify-content-center`

Comment (JubalSantos) [reply resources/views/database/links.blade.php:67]: this is an example of doing it <h2 class="text-center"> name</h2>

Comment (giangdavis) [reply resources/views/database/links.blade.php:67]: ty

**Pull request merged**